src/commands/deploy.ts

```diff
@@ -664,10 +664,10 @@ export async function performCloudProviderABTesting(
     const isAlreadyDeployed = await isProjectDeployed(projectName, projectRegion);
 
     // We won't perform AB testing if the cloud provider is set for runtime, self-hosted or cluster.
-    if (!isAlreadyDeployed && projectCloudProvider === CloudProviderIdentifier.GENEZIO) {
+    if (!isAlreadyDeployed && projectCloudProvider === CloudProviderIdentifier.GENEZIO_AWS) {
```

Comment:
Also, in case it is not already deployed, I think we should prevent using the legacy values.

Reply:
I've added a clear warning in case a legacy provider is used.